dmutils/email/dm_mailchimp.py

```diff
@@ -161,3 +163,35 @@ def get_email_addresses_from_list(self, list_id, pagination_size=100):
             offset += pagination_size
 
             yield from [member['email_address'] for member in member_data['members']]
+
+    def get_lists_for_email(self, email_address: str) -> Sequence[Mapping]:
+        """
+            Returns a sequence of all lists the email_address has an association with (note: even if that association is
+            "unsubscribed" or "cleaned").
+        """
+        with log_external_request(service='Mailchimp'):
+            return tuple(
+                {
+                    "list_id": ml["id"],
+                    "name": ml["name"],
+                } for ml in self._client.lists.all(get_all=True, email=email_address)["lists"]
```

Comment:
I'm not sure what an `ml` is. Could you use a more descriptive variable name?

Reply:
ml, "mailing list", thought that was pretty universal.

Reply:
I don't think it is. Could you change it?